src/hooks/useCombinedRefs.ts

```diff
@@ -6,7 +6,7 @@ export function useCombinedRefs<T>(...refs: readonly React.Ref<T>[]) {
       for (const ref of refs) {
         if (typeof ref === 'function') {
           ref(handle);
-        } else if (ref !== null) {
+        } else if (ref !== null && 'current' in ref) {
```

Comment:
Without `forwardRef`, the `ref` parameter is actually an empty, frozen object, and trying to assigning `current` to it crashes.